app/src/main/java/com/omelan/cofi/MainActivity.kt

```diff
@@ -101,6 +102,25 @@ class MainActivity : MonetCompatActivity() {
         }
     }
 
+    private fun isAppInstalled(packageName: String): Boolean {
+        return try {
+            packageManager.getPackageInfo(packageName, 0)
+            true
+        } catch (e: Exception) {
+            false
+        }
+    }
+
+    private fun showInstallPrompt() {
+        // Here it shows a install prompt for instant apps
+        val postInstall = Intent(Intent.ACTION_MAIN)
+            .addCategory(Intent.CATEGORY_DEFAULT)
+            .setPackage("com.omelan.cofi")
```

Comment:
This shouldn't be hardcoded
```suggestion
            .setPackage(BuildConfig.APPLICATION_ID)
```